schema/remotehosts.json

```diff
@@ -140,7 +140,21 @@
                 "userenv": {
                     "type": "string",
                     "minLength": 1
-                }
+                },
+		"podman-settings": {
+                    "type": "object",
+		    "properties": {
+                        "device": {
+                            "type": "string",
+                            "minLength": 1
+                        },
+                        "shm-size": {
+                            "type": "string",
+                            "minLength": 1
+                        }
```

Comment:
Would it be possible to make the validations of these properties any more accurate or are they really completely arbitrary strings?

Reply:
I think over time it would be a good idea to have these match something like a regex.  I am not yet familiar with how to do that in the json validator.  Or maybe better suited for endpoint validation?